README.md

```diff
@@ -17,6 +17,7 @@ URL Builder
 ===========
 
 ## This library is now deprecated.
+This library is no longer in usage you need to implement your own code to use this service.
```

Comment:
Does not quite make sense, it's no longer in use, but probably no need for the last part of the line.

Reply:
sure, I will remove the last part.